diff -ur a/util/grub-mkconfig_lib.in b/util/grub-mkconfig_lib.in
--- a/util/grub-mkconfig_lib.in	2016-07-08 15:45:13.810000000 +0200
+++ b/util/grub-mkconfig_lib.in	2016-07-08 15:46:19.470000000 +0200
@@ -145,7 +145,7 @@
   done
 
   if [ x$GRUB_ENABLE_CRYPTODISK = xy ]; then
-      for uuid in "`"${grub_probe}" --device $@ --target=cryptodisk_uuid`"; do
+      for uuid in `"${grub_probe}" --device $@ --target=cryptodisk_uuid`; do
 	  echo "cryptomount -u $uuid"
       done
   fi
